agr_java_core/src/main/java/org/alliancegenome/core/translators/tdf/AlleleToTdfTranslator.java

```diff
@@ -253,9 +253,13 @@ public String getAllVariantsRows(List<Variant> variants) {
     }
 
 
-    public List<VariantDownloadRow> getVariantDownloadRowsForAlleles(List<Variant> annotations) {
-
+    public List<VariantDownloadRow> VariantDownloadRowsForAlleles(List<Variant> annotations) {
```

Comment:
sorry to be picky but could we use maybe `convertVariantDownloadRowsForAlleles` ?

Reply:
or `translateVariantDownloadRowsForAlleles`

Reply:
I don't mind the original method name too much other than it should use 'Variant' instead of Alleles the entities passed into the method. It's a translator method and so methods are facilitating a translation from one entity to another. 
getVariantDownloadRowsForVariants.
From the signature it becomes clear that you are turning a list of variants into a list or VariantDownloadRows. that does not need to be encoded into the method name.